pom.xml

```diff
@@ -141,7 +141,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.20</version>
+            <!-- Snapshot for better handling of onLoad errors -->
+            <version>2.21-20180427.223321-5</version>
```

Comment:
[Pro tip](https://github.com/jenkinsci/pom/blob/master/incrementals.md#usage-in-plugin-poms)

Reply:
@jglick Since you only added that a few weeks ago and thus it's not "obvious" maybe you meant something [besides 'pro tip'](https://www.urbandictionary.com/define.php?term=Protip)?